src/components/kubectl/logs.ts

```diff
@@ -113,19 +115,20 @@ async function getLogsForContainer(
     const panel = LogsPanel.createOrShow('Loading...', resource);
 
     if (displayMode === LogsDisplayMode.Follow) {
-        cmd = `${cmd} -f`;
-        kubectl.invokeAsync(cmd, undefined, (proc: ChildProcess) => {
-            proc.stdout.on('data', (data: string) => {
-                panel.addContent(data);
-            });
-        });
+        const lines = await kubectl.observeCommand(args);
+
+        lines.subscribe(
+            (line) => { if (line) { panel.addContent(`${line}\n`); } },
```

Comment:
Here we have the same issue we have now. The process remains alive even if the log panel is disposed and it keeps calling the `addContent` method. What worries me is not the error itself because this PR is not related to that one and it can be fixed in a second moment but i am not sure the observeCommand gives us a reference to keep track of the childProcess created (???)

Reply:
Good catch - thank you.  I'll update `observeCommand` to track the child process.  I expect your fix to merge first, so will need to update the logs stuff in this PR once I rebase.

Reply:
Grr.  `spawnrx` just gives us the output, not the process object.  But the docs say that unsubscribing from the output stream terminates the process, so I think we can mimic this closely enough.